committee-steering/governance/ug-governance.md

```diff
@@ -0,0 +1,37 @@
+# Kubernetes User Group (UG)
+
+## Introduction
+The Kubernetes community and ecosystem benefits providing a place for communication and collaboration between users and community members on topics that either do not have deliverables or are outside the scope of the project itself. These topics are not appropriate for forming SIGs, WGs or Committees, which are intended to drive work within the project. User Groups as the venue for this kind of communication and collaboration.
+
+Example User Groups:
+
+- Big data - Topics related to Big Data integrations.
+- Public Cloud - Topics related to public cloud integrations.
```

Comment:
Can we reword `Public Cloud` to `Cloud Providers` since we anticipate user groups for private clouds as well :)